### PR TITLE
1.21.30 support

### PR DESCRIPTION
**Pull request by pokecosimo**

### CODE REVIEW

File: tools/js/test/audit_blockitems.js
Comment (extremeheat) [inline tools/js/test/audit_blockitems.js:1]: please revert file deletions

Comment (pokecosimo) [reply tools/js/test/audit_blockitems.js:1]: when i get home, ill'do that

Comment (pokecosimo) [reply tools/js/test/audit_blockitems.js:1]: done

**Review by NoVa-Gh0ul — APPROVED** [review]

**Comment by extremeheat** [comment]
Thanks for working on this. There were several mistakes made here so it took a while longer to review and fix, but I believe I've fixed most of the issues.

**Comment by pokecosimo** [comment]
> Thanks for working on this. There were several mistakes made here so it took a while longer to review and fix, but I believe I've fixed most of the issues.

Sorry for my errors, first time doing it

**Comment by extremeheat** [comment]
Thanks for the work on this

**Pull request merged**